Cargo.toml

```
@@ -14,6 +14,7 @@ edition = "2021"
 # and enable individual features in indiv crates
 borsh = { version = "^1", default-features = false }
 bs58 = { version = "^0.5", default-features = false }
+bs58-fixed-wasm = { git = "https://github.com/igneous-labs/bs58-fixed.git", default-features = false }
```

we should always be explicit about which branch or commit git deps are using
```suggestion
bs58-fixed-wasm = { git = "https://github.com/igneous-labs/bs58-fixed.git", branch = "master", default-features = false }
```